alpha/apps/kaltura/lib/kEntryPlayingDataHelper.php

```diff
@@ -0,0 +1,443 @@
+<?php
+class kEntryPlayingDataHelper
+{
+	const MEDIA_PROTOCOL_HTTP = 'http';
+	const MEDIA_PROTOCOL_HTTPS = 'https';
+
+	/**
+	 * @param $dbEntry
+	 * @param $partner
+	 * @param $contextDataParams
+	 * @return kContextDataHelper
+	 */
+	public function initContextDataHelper($dbEntry, $partner, $contextDataParams)
+	{
+		$asset = null;
+		if ($contextDataParams->flavorAssetId)
+		{
+			$asset = assetPeer::retrieveById($contextDataParams->flavorAssetId);
+			if (!$asset)
+				throw new KalturaAPIException(KalturaErrors::FLAVOR_ASSET_ID_NOT_FOUND, $contextDataParams->flavorAssetId);
```

Comment:
KalturaErrors is API class, use APIErrors in the core.
Same for KalturaAPIException.